src/test/java/hw5/pages/SecondFrame.java

```diff
@@ -0,0 +1,21 @@
+package hw5.pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class SecondFrame extends BaseElements {
+    private WebDriver driver;
```

Comment:
Why do you need this field here>?

Reply:
To initialize homepage on method switchToParent(). I can replace the type of the returned parameter with a void and don't use the driver.

Reply:
I can't remove because I use driver: driver.switchTo().parentFrame();